src/main/java/com/eternalcode/combat/fight/event/FightUntagEvent.java

```diff
@@ -0,0 +1,29 @@
+package com.eternalcode.combat.fight.event;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+public class FightUntagEvent extends Event {
+
+    private static final HandlerList handlers = new HandlerList();
+    private final Player player;
```

Comment:
aby ograniczyć używanie rzeczy typu Bukkit.getPlayer itd zmień playera na UUID.
później jak będziesz chciał pobrać playera to użyj Server#getPlayer

Reply:
using Bukkit.getPlayer(event.getPlayer()) is okay?